ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -236,39 +240,28 @@ export default function CreatePassword({
 
   return (
     <div className="create-password__wrapper" data-testid="create-password">
-      {
-        ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
-        secretRecoveryPhrase &&
-        firstTimeFlowType === FIRST_TIME_FLOW_TYPES.IMPORT ? (
-          <TwoStepProgressBar
-            stage={twoStepStages.PASSWORD_CREATE}
-            marginBottom={4}
-          />
-        ) : (
-          <ThreeStepProgressBar
-            stage={threeStepStages.PASSWORD_CREATE}
-            marginBottom={4}
-          />
-        )
-        ///: END:ONLY_INCLUDE_IN
-      }
-
-      <Typography variant={TypographyVariant.H2} fontWeight={FONT_WEIGHT.BOLD}>
+      {secretRecoveryPhrase &&
```

Comment:
It appears that the build type comments are currently missing in this code, and we'd like to retain them. These comments serve the purpose of conditionally adding or removing code based on the specific build type.
```suggestion
      {
        ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
        secretRecoveryPhrase &&
```